source/math/FloatVectorList.ooc

```diff
@@ -212,4 +212,33 @@ FloatVectorList: class extends VectorList<Float> {
 		previous free()
 		result
 	}
+	median: func -> Float {
+		result: Float
+		tempVector := this copy()
+		tempVector sort()
```

Comment:
Also, there are ways of finding the median that do not require sorting the data, but I don't expect you to have time to implement one. Here's one in C++, though: http://discuss.codechef.com/questions/1489/find-median-in-an-unsorted-array-without-sorting-it